ChangeLog.md

```diff
@@ -1,3 +1,75 @@
+* If game picks gold for a random resource pile that has predetermined by map amount, its amount will be correctly multiplied by 100
+* Fixed hero not being able to learn spells from a mod in some cases, even if they are available from the town's mage guild.
+* The game will now actually take resources from seers' huts with the Gather Resources mission instead of awarding them.
+* Heroes with double spell points will no longer trigger the Mana Vortex.
+
+### Interface
+* Fixed reversed button functions in Exchange Window
+* Fixed allied towns being missing from the list when using the advanced or expert Town Portal spell.
+* Fixed corrupted UI that could appear for a frame under certain conditions
+* The '*' symbol and non-printable characters can no longer be used in savegames due to Windows file system restrictions.
+* Pressing Ctrl while hovering over the adventure map will now display tile coordinates in the status bar.
+* Selection of another hero while hero is selected now requires Shift press instead of Ctrl
+* Fixed hero troops in the info box view flashing briefly during hero movement.
+* Reduced excessive memory usage on adventure map by several hundreds of megabytes (most noticeable on systems with large screen resolution)
+* Haptic feedback is now enabled by default on Android and on iOS
+* It is now possible to scroll through artifacts backpack using mouse wheel or swipe
+
+### Launcher
+* Android now uses the same Qt-based launcher as other systems
+* Fixed attempt to install a submod when installing new mod that depends on a submod of another mod
+* Fixed wrong order of activating mods in chain when installing multiple mods at once
+* Mod list no longer shows mod version column. Version is now only shown in the mod description.
+* Launcher will now skip the Heroes 3 data import step if data has been found automatically
+* Fixed inport of existing data files on iOS. This option now requires iOS 13 or later
+* Fixed import using offline installer on iOS.
+* Buttons to open data directories in the Help tab are now hidden on mobile systems if they can't be opened with file browser
+* Added the configuration files directory to the Help tab as it is located separately on Linux systems
+* Removed H3 data language selection during setup in favor of auto-detection
+* Replaced checkboxes with toggle buttons for easier of access on touchscreens.
+* Added interface for configuring several previously existing but inaccessible options in Launcher:
+* * Selection of input tolerance precision for all input types
+* * Relative cursor mode for mobile systems (was only available on Android)
+* * Haptic feedback toggle for mobile systems (was only available on Android)
+* * Sound and music volume (was only available in game)
+* * Selection of long touch interval (was only available in game)
+* * Selection of upscaling filter used by SDL 
+* * Controller input sensitivity and acceleration.
```

Comment:
```suggestion
    * Selection of input tolerance precision for all input types
    * Relative cursor mode for mobile systems (was only available on Android)
    * Haptic feedback toggle for mobile systems (was only available on Android)
    * Sound and music volume (was only available in game)
    * Selection of long touch interval (was only available in game)
    * Selection of upscaling filter used by SDL 
    * Controller input sensitivity and acceleration.
```

The two asterisks are rendered like this:

![image](https://github.com/vcmi/vcmi/assets/3226457/bf59dad6-eb22-4a6d-9dae-953905f76db9)